### PR TITLE
[Web Portal] Add tooltips to job submission page's field label.

### PR DESCRIPTION
**Pull request by sunqinzheng**

### CODE REVIEW

File: src/webportal/src/app/job-submission/utils/constants.jsx [hunk 1]
```diff
@@ -74,3 +76,23 @@ export const PAI_ENV_VAR = [
...
+    <div>
+      <p>Resource required per container instance</p>
+      <p>CPU number and memory number will be auto scaled with GPU number by default</p>
+    </div>
+  ),
+  taskRoleCPU: 'CPU number, unit is CPU vcore',
+  taskRoleMemory: 'Memory number, unit is MB',
+  taskRoleGPU: 'GPU number, unit is GPU card',
+  taskRoleDocker: `Container's docker image`,
+  jobInformation: 'Job Information',
```
Comment (sunqinzheng) [inline src/webportal/src/app/job-submission/utils/constants.jsx]: I don't know what should be displayed here. Any suggestions?

File: src/webportal/src/app/job-submission/components/job-information.jsx [hunk 2]
```diff
@@ -62,9 +62,26 @@ export const JobInformation = React.memo(({jobInformation, onChange, advanceFlag
...
+              gapSpace: 8, // spacing.s1
+            }}
+            content='Click to open job help page'
+          >
+            <IconButton
+              iconProps={{iconName: 'HintText'}}
+              styles={{root: {height: '100%'}}}
+              href='https://github.com/microsoft/pai/blob/master/docs/pai-job-protocol.yaml'
+              target='_blank'
+              text='asdasdads'
```
Comment (abuccts) [inline src/webportal/src/app/job-submission/components/job-information.jsx]: remove?

File: src/webportal/src/app/job-submission/utils/constants.jsx (hunk 1 above)
Comment (abuccts) [reply src/webportal/src/app/job-submission/utils/constants.jsx]: "Job Spec"?
"Click to view job spec details"

File: src/webportal/src/app/job-submission/components/job-information.jsx (hunk 2 above)
Comment (sunqinzheng) [reply src/webportal/src/app/job-submission/components/job-information.jsx]: sorry, I'll remove this line

File: src/webportal/src/app/job-submission/utils/constants.jsx (hunk 1 above)
Comment (sunqinzheng) [reply src/webportal/src/app/job-submission/utils/constants.jsx]: Job Information's tooltip is changed to a `learn more` link

**Pull request merged**